src/illustrativeIcons/icon-illustration-global.svg

```diff
@@ -1,42 +1,38 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <svg width="80px" height="80px" viewBox="0 0 80 80" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <!-- Generator: Sketch 49.3 (51167) - http://www.bohemiancoding.com/sketch -->
-    <title>Ikonit / Kuvitukset / Kansainvälistyminen</title>
-    <desc>Created with Sketch.</desc>
-    <defs></defs>
-    <g id="Ikonit-/-Kuvitukset-/-Kansainvälistyminen" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+    <title>Icons/Illustrative Icons/Internationalization</title>
+    <g id="Icons/Illustrative-Icons/Internationalization" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
         <g id="Kansainvälistyminen" transform="translate(8.000000, 8.000000)">
```

Comment:
Just a side note, but is it possible that Scandinavian characters might cause troubles in the svgs?

Reply:
Should not be possible. Uses utf-8 characterset.

Reply:
And those id's are never referred by anyone, but in many cases are not cleaned out by compilation.